src/main/scala/chisel3/tester/internal/Testers2.scala

```diff
@@ -25,6 +29,47 @@ case class TesterOptions(
   }
 }
 
+trait TestOption extends Unserializable { this: Annotation => }
+trait TestOptionObject extends NoTargetAnnotation with HasShellOptions with TestOption
+
+case object WriteVcdAnnotation extends TestOptionObject {
```

Comment:
Does this get used anywhere - I see an import treadle.WriteVcdAnnotation remaining elsewhere in the code? Also, why isn't this in Treadle?

Reply:
I am not entirely happy with this; here is my reasoning for keeping it. Treadle has and uses a WriteVcd annotation but verilator uses a SuppressVcd. I think there will be a few examples like this where similar concepts are implemented differently by different backends and in those cases testers2 may have an annotation for the high-level construct that gets mapped to the backend specific version.  Your thoughts?

Reply:
I think since we have control over the backends and how they interpret annotations, we should be able to standardize it. Perhaps there should be a collection of standard simulator annotations that backends should understand?

I'm fine with legacy backends (like pre-refactor Verilator) being spotty but we should keep the modern backends (treadle, post-refactor Verilator) compliant with whatever standard annotations we set.

Reply:
I'd probably like to think about this, but I expect we could do a common `Stage` here and then subclass off that. I agree that unifying things as much as possible would be great for a future non-legacy version.

Reply:
I have changed verilator now to behave the same treadle. The default for both is no VCD is produced. Adding a testers2 WriteVcdAnnotation causes VCD output to be turned on for both.
The internal mechanisms are different between the backends and TreadleBackend converts the testers2 WriteVcdAnnotation into a treadle WriteVcdAnnotation. As new backends are added, the <NewBackend> and <NewBackendExecutive> classes will mediate when necessary between any global testers2 general options and their own internal representations. I'm happy to discuss this more but I feel ok about the current implementation until we have more backends